script.js

```diff
@@ -1,64 +1,207 @@
 var player = 1;
 var jogadas = 0;
 var ha_vencedor = false;
-var valida_posicao =  new Array(0,0,0,0,0,0,0,0,0);
-var marca_posicao = new Array();
 
-function clic(posicao_jogada){
+var marca_posicao =  [
+    [false,false,false],
+    [false,false,false], 
+    [false,false,false], 
+];
+
+document.getElementById('vez').innerHTML = 'Vez de O';
+document.getElementById('jogadas').innerHTML ='Jogadas feitas: '+jogadas;
+
+function getRandomInt(min, max) {
+    min = Math.ceil(min);
+    max = Math.floor(max);
+    return Math.floor(Math.random() * (max - min)) + min;
+}
+var linhaRandom = getRandomInt(0,2);
+var colunaRandom = getRandomInt(0,2);
+
+function insereX(linha,coluna){
+    document.getElementById('pos'+linha+coluna).style.backgroundImage = 'url("images/xis.jpg")';
+}
+
+function play(linha,coluna){
     
-    if(valida_posicao[posicao_jogada] == 0 && ha_vencedor == false){
+    if(marca_posicao[linha][coluna] == false && ha_vencedor == false){
         if(player == 1) {
-            valida_posicao[posicao_jogada] = "bola";
-            document.getElementById("pos"+posicao_jogada).style.backgroundImage = 'url("images/bola.jpg")';
-            marca_posicao[posicao_jogada] = 'O';
+            marca_posicao[linha][coluna] = 'bola';
+            document.getElementById('pos'+linha+coluna).style.backgroundImage = 'url("images/bola.jpg")';
```

Comment:
Why not creating a function called `insertO` just like you did for `insereX`?

Or even better. You can make it even more generic:
```js
function insertChar(line, column, char) {
    var image;
    if (char === 'O') {
        image = 'url("images/bola.jpg")';
    } else if (char === 'X') {
        image = 'url("images/xis.jpg")';
    }

    document.getElementById('pos' + line + column).style.backgroundImage = image;
    marca_posicao[line][column] = char;
}
```